aws/ecs/task_execution_roles.tf

```diff
@@ -31,7 +31,12 @@ resource "aws_iam_role_policy_attachment" "te_etl_policies" {
   policy_arn = aws_iam_policy.etl_policies.arn
 }
 
+resource "aws_iam_role_policy_attachment" "secretsmanager_etl_policies" {
+  role       = aws_iam_role.task_execution_role.name
+  policy_arn = aws_iam_policy.get_metrics_token_secret_value_ecs_task.arn
+}
+
```

Comment:
Instead of attaching this to the task execution role that's used by the in-app metrics we should probably create a role just for the server metrics.